### PR TITLE
codegen/protobuf: drop unnecessary code generation scripts

### PR DESCRIPTION
**Pull request by fmuyassarov**
Currently this repo doesn't have any CRD go type definitions nor the protobuf
for gRPC. As such, clean up all the bits related to CRD and protobuf code
generation.
/cc @jukkar @klihub PTAL

### CODE REVIEW

**Comment by fmuyassarov** [comment]
Please don't merge this yet, I will add protobuf related bits clean up too as discussed.

**Comment by klihub** [comment]
@fmuyassarov We converted it to a draft as an indication of the PR to not be ready for merging. Mark it ready again once you'd like it to be merged.

**Comment by fmuyassarov** [comment]
@jukkar @klihub I've pushed a second commit for the protobuf related code clean up. PTAL, thanks.

File: Dockerfile_generator
Comment (jukkar) [inline Dockerfile_generator]: Do we need the Dockerfile_generator any more if we do not have the code generation?

File: Makefile
Comment (jukkar) [inline Makefile]: I think we can remove these lines too.

File: Dockerfile_generator
Comment (fmuyassarov) [reply Dockerfile_generator]: I actually don't know why we need that file for. If you think there is no use, I can remove it too.

Comment (fmuyassarov) [reply Dockerfile_generator]: ah right, it was for generating code inside the container like in NFD. yes, then we should remove it too.

**Review by klihub — CHANGES_REQUESTED** [review]
There is a typo ('releted' vs. 'related') in the other commit's commit message.

**Comment by klihub** [comment]
@fmuyassarov @jukkar Other than the typo, if we go with multiple commits (for nuking CRD generation and code generation from protobufs) , let's try to structure them so that by reverting one (sans possible inevitable conflicts) we restore the ability to generate CRDs and by reverting the other (sans possible inevitable conflicts) we restore the ability to generate code from protobufs... if possible without too much effort.

**Comment by fmuyassarov** [comment]
> @fmuyassarov @jukkar Other than the typo, if we go with multiple commits (for nuking CRD generation and code generation from protobufs) , let's try to structure them so that by reverting one (sans possible inevitable conflicts) we restore the ability to generate CRDs and by reverting the other (sans possible inevitable conflicts) we restore the ability to generate code from protobufs... if possible without too much effort.

Yes that can be done, but this dockerfile_generator is shared between crd and protobuf. So I'm wondering on which of the commit I shall remove that.

**Comment by fmuyassarov** [comment]
> There is a typo ('releted' vs. 'related') in the other commit's commit message.

sorry and thanks for the notice. Fixed.

**Comment by jukkar** [comment]
> Yes that can be done, but this dockerfile_generator is shared between crd and protobuf. So I'm wondering on which of the commit I shall remove that.

Hmm, sounds like that perhaps we should just merge the two commits together, would that be a big problem later when/if we revert the commit?

**Comment by klihub** [comment]
> > Yes that can be done, but this dockerfile_generator is shared between crd and protobuf. So I'm wondering on which of the commit I shall remove that.
> 
> Hmm, sounds like that perhaps we should just merge the two commits together, would that be a big problem later when/if we revert the commit?

True, it is probably the simplest way to go.

@fmuyassarov Can you squash the commits to a single one and update the commit message accordingly ?

**Comment by fmuyassarov** [comment]
@jukkar @klihub I squashed the commits and updated the commit message accordingly. PTAL

**Review by klihub — APPROVED** [review]
LGTM.

**Review by jukkar — APPROVED** [review]
LGTM

**Pull request merged**
